### PR TITLE
[IEXPLORE] Update now-redirected 'http' and 'www.reactos.org' URLs

### PR DESCRIPTION
**Pull request by SergeGautherie**

### CODE REVIEW

File: base/applications/iexplore/iexplore.inf [hunk 1]
```diff
@@ -28,11 +28,11 @@ HKCR,"CLSID\%CLSID_InternetExplorerManager%\LocalServer32",,,"""%16422%\Internet
 [Settings.Reg]
 HKCU,"Software\Microsoft\Internet Explorer\Main","ShowGoButton",2,"yes"
 #HKCU,"Software\Microsoft\Internet Explorer\Main","Start Page",2,"https://www.winehq.org"
-HKCU,"Software\Microsoft\Internet Explorer\Main","Start Page",2,"http://www.reactos.org"
+HKCU,"Software\Microsoft\Internet Explorer\Main","Start Page",2,"https://reactos.org/"
 HKCU,"Software\Microsoft\Internet Explorer\Main","Search Page",2,"http://www.google.com"
```
Comment (KyleKatarn31) [inline base/applications/iexplore/iexplore.inf:32]: should we consider using HTTPS for Google as search page too ?

Comment (SergeGautherie) [reply base/applications/iexplore/iexplore.inf:32]: That is off-topic in this PR. Make your own PR, if need be...

Comment (KyleKatarn31) [reply base/applications/iexplore/iexplore.inf:32]: Not needed. Disregard.

Comment (SergeGautherie) [reply base/applications/iexplore/iexplore.inf:32]: Filed as #2646.

**Pull request merged**